### PR TITLE
Extracted eth_call to helper to support polling.

### PR DESCRIPTION
**Pull request by ebadiere**
**Description**:
Adds polling to avoid using hardcoded delays.

**Related issue(s)**: #2013 

Fixes #

### CODE REVIEW

File: packages/server/tests/helpers/utils.ts [hunk 1]
```diff
@@ -150,4 +152,19 @@ export class Utils {
     }
     return res.toString();
   };
+
+  static ethCall = async (
+    relay: RelayClient,
+    callData: { from: string; to: any; gas: string; data: string },
+    requestId: string,
```
Comment (Nana-EC) [inline packages/server/tests/helpers/utils.ts:159]: Add block as a param, right now you've defaulted to `latest` with no way to modify

File: packages/server/tests/helpers/utils.ts [hunk 2]
```diff
@@ -150,4 +152,19 @@ export class Utils {
     }
     return res.toString();
   };
+
+  static ethCall = async (
```
Comment (Nana-EC) [inline packages/server/tests/helpers/utils.ts:156]: Update name to make clear there's a retry e.g ethCallWRetries or something.
We want to make sure people use this method only when they want retries

**Review by Nana-EC — COMMENTED** [review]
Looking good, some suggestions

File: packages/server/tests/helpers/utils.ts (hunk 1 above)
Comment (ebadiere) [reply packages/server/tests/helpers/utils.ts:160]: Done.

File: packages/server/tests/helpers/utils.ts (hunk 2 above)
Comment (ebadiere) [reply packages/server/tests/helpers/utils.ts:157]: Done.

File: packages/server/tests/helpers/utils.ts [hunk 3]
```diff
@@ -150,4 +153,39 @@ export class Utils {
...
+      await new Promise((r) => setTimeout(r, 2000));
+      res = await relay.call(RelayCall.ETH_ENDPOINTS.ETH_CALL, [callData, blockNumber], requestId);
+      numberOfCalls++;
+    }
+    return res;
+  };
+
+  // Handles the odd case where the contract is loading in the factory but for some reason
+  // ethers is not able to deploy it.  The ethers deploy returns a 4001 error.
+  static deployContract = async (contractFactory: ContractFactory): Promise<ethers.Contract> => {
```
Comment (Nana-EC) [inline packages/server/tests/helpers/utils.ts:175]: This method will indefinitely retry under the right circumstances.
You should limit the number if retries and update the method name appropriately.

**Review by Nana-EC — CHANGES_REQUESTED** [review]

File: packages/server/tests/helpers/utils.ts (hunk 3 above)
Comment (ebadiere) [reply packages/server/tests/helpers/utils.ts:179]: Fixed.

**Comment by ebadiere** [comment]
Rebased too many changes.  Cherry-picked relevant commits into a new PR: https://github.com/hashgraph/hedera-json-rpc-relay/pull/2031

**Pull request closed without merging**